components/dropbox/actions/create-or-append-to-a-text-file/create-or-append-to-a-text-file.mjs

```diff
@@ -0,0 +1,86 @@
+import dropbox from "../../dropbox.app.mjs";
+import common from "../../common.mjs";
+
+export default {
+  name: "Create or Append to a Text File",
+  description: "Adds a new line to an existing text file, or creates a file if it doesn't exist. [See docs here](https://dropbox.github.io/dropbox-sdk-js/Dropbox.html#filesUpload__anchor)",
+  key: "dropbox-create-or-append-to-a-text-file",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    dropbox,
+    name: {
+      type: "string",
+      label: "File name",
+      description: "Your new file name.",
+    },
+    path: {
+      propDefinition: [
+        dropbox,
+        "pathFolder",
+      ],
+      optional: true,
+      description: "The Path in the user's Dropbox to create or append in the file. If not filled, it will be created on the root folder. (Please use a valid path to filter the values)",
+    },
+    content: {
+      type: "string",
+      label: "Content",
+      description: "The content to be written",
+    },
+  },
+  methods: {
+    ...common.methods,
+    async getFileInfo(path, content) {
+      let file;
+      try {
+        file = await this.dropbox.downloadFile({
+          path,
+        });
+      } catch {
+        file = null;
+      }
```

Comment:
I couldn't get this action to work because the `path` isn't including the filename. It works if I add this line 
```path = `${path}${this.name}`;```